jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -629,6 +629,12 @@
         jenkins-timeout: 170
         timeout: 70
         trigger-job: 'ci-kubernetes-build'
+    - kubernetes-e2e-gce-gci-qa-m60:
```

Comment:
nitpick: why not add this new stanza in between `master` and `m59` for consistency, here and elsewhere in this file?